[Keyboard] Add new keyboard ks63 (#9712)
qmk/qmk_firmware: keyboards/handwired/ks63/README.md

> # ks63

Reviewer: Please rename this file to lowercase.

Author: ok

![ks63](https://github.com/klesh/ks63/raw/master/images/ks63-finish-02.jpg)

A handwired 63 keys split keyboard compatible with Default 60% layout. Let you keep your magic power while muggles can still use it as an ordinary keyboard

* Keyboard Maintainer: [Klesh Wong](https://github.com/klesh)
* Hardware Supported: Arduino ProMicro
* Assembling Guide: [ks63](https://github.com/klesh/ks63)

> Compile with qmk cli (after setting up your build environment):
> 
>     qmk compile -kb handwired/ks63 -km default

Reviewer: ```suggestion
Make example for this keyboard (after setting up your build environment):

    make handwired/ks63:default
```

> [Please checkout here](https://github.com/klesh/ks63)
> 
> # ks63

Reviewer 2: ```suggestion
# ks63
```
